tests/.env

```diff
@@ -5,3 +5,8 @@ AZURE_SUBSCRIPTION=
 AZURE_RESOURCE_GROUP=
 TEST_LOG_ANALYTICS_WORKSPACE_ID=
 TEST_STORAGE_CONNECTION_STRING=
+OS_SWIFT_URL=
+OS_TENANT_ID=
+OS_PASSWORD=
+OS_PROJECT_ID=
+OS_AUTH_URL=
```

Comment:
We will need to fill these values right?

Reply:
The test will always look for these environment variables to run. I thought that every env variable used by tests should be declared there. Am I right or is it not necessary?

Reply:
Yeah, the question was more about the OpenStack instance itself. Gonna ping you on Slack